Menu/Menu.js

```diff
@@ -44,29 +44,28 @@ function newItem(arr) {
 
```

Comment:
good job, and good separator comments, try and add some comments for details/explanations too